### PR TITLE
Resolves Issue No #118 #132 #116 | Scroll Bar Fixed

### PR DESCRIPTION
**Pull request by ashutoshsaboo**
Resolves Issues #118 #132 #116 . Scroll Bar bug fixed. 

This PR is continued after #135 . Since there were some conflicts in the branch in #135 , hence I created this new PR. 

@SaptakS @joerx @daminisatya @mariobehling @jig08 @AmayAgrawal @championswimmer @creativepsyco Please have a look at this PR. 

@championswimmer I have fixed the fixes, that you suggested in #135 .

### CODE REVIEW

**Comment by ashutoshsaboo** [comment]
@championswimmer So, this has also passed all the tests now. Can this be merged now?

**Comment by ashutoshsaboo** [comment]
Also, this branch doesn't have any conflicts. So, is this good to merge now? @championswimmer

**Comment by ashutoshsaboo** [comment]
@championswimmer Can this be merged now?

File: app/components/speakers/speakers.html [hunk 1]
```diff
@@ -57,4 +58,9 @@
         </md-list>
         <div layout="column" flex style="align-content: center"> <p align="center"> <md-progress-circular ng-if="sc.showLoaders" md-mode="indeterminate"></md-progress-circular></p></div>
     </md-content>
+<<<<<<< HEAD
```
Comment (championswimmer) [inline app/components/speakers/speakers.html]: clean the conflicts

**Comment by ashutoshsaboo** [comment]
Hi @aayusharora . I ran these set of commands-:

```
ashutoshsaboo@ASHUTOSH-PC:~/open-event-webapp$ git checkout master 
Switched to branch 'master'
Your branch is up-to-date with 'origin/master'.
ashutoshsaboo@ASHUTOSH-PC:~/open-event-webapp$ git pull origin master 
From https://github.com/ashutoshsaboo/open-event-webapp
 * branch            master     -> FETCH_HEAD
Already up-to-date.
ashutoshsaboo@ASHUTOSH-PC:~/open-event-webapp$ git checkout changes3 
Switched to branch 'changes3'
ashutoshsaboo@ASHUTOSH-PC:~/open-event-webapp$ git merge master 
Already up-to-date.
ashutoshsaboo@ASHUTOSH-PC:~/open-event-webapp$ git pull origin changes3 
From https://github.com/ashutoshsaboo/open-event-webapp
 * branch            changes3   -> FETCH_HEAD
Already up-to-date.
ashutoshsaboo@ASHUTOSH-PC:~/open-event-webapp$ git branch 
  changes
* changes3
  master
ashutoshsaboo@ASHUTOSH-PC:~/open-event-webapp$ git status 
On branch changes3
nothing to commit, working directory clean

```

Everything seems to be updated with the origin/master, then why is it giving conflict error? Could you help me resolve this? @aayusharora

**Comment by aayusharora** [comment]
@ashutoshsaboo ,it is because you are pulling changes from the origin https://github.com/ashutoshsaboo/open-event-webapp , you should pull changes from the remote ( by adding the remote from where you have forked).
After that remove the conflicts on your local and push it to your origin.

**Pull request closed without merging**